packages/trader/src/AppV2/Utils/contract-details-config.ts

```diff
@@ -0,0 +1,45 @@
+import { CONTRACT_TYPES } from '@deriv/shared';
+
+const DEFAULT_CONFIG = {
+    is_take_profit_visible: false,
+    is_deal_cancellation_visible: false,
+    is_stop_loss_visible: false,
+    is_tp_history_visible: false,
+};
+
+export const getContractDetailsConfig = (contract_type: string) => {
```

Comment:
If it's used mainly for Risk Management stuff, maybe it will be good idea to rename this config accordantly?

Reply:
I think it can be used for other things in the future, so I'd keep generic name for it for now